tests/kani/Strings/copy_empty_string_by_intrinsic.rs

```diff
@@ -21,11 +29,18 @@ fn copy_string(s: &str, l: usize) {
         // Copy
         let src = from_raw_parts(s.as_ptr(), l).as_ptr();
         copy_nonoverlapping(src, dest, l);
+
+        // The chunk below causes the 3 failures at the top of the file
+        // Back to str, check length
+        let dest_slice: &[u8] = from_raw_parts(dest, l);
+        let dest_as_str: &str = str::from_utf8(dest_slice).unwrap();
+        assert!(dest_as_str.len() == l);
     }
 }
 
 #[kani::proof]
 fn main() {
+    // Verification fails for both of these cases.
```

Comment:
Looks like these are passing now. Is this comment needed?

Reply:
Indeed. I can update the comments.